src/js/view/components/listmenu.js

```diff
@@ -0,0 +1,57 @@
+define([
```

Comment:
Too much duplication here. Just make non-togglable a setup configuration option in menu.js.